server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -160,6 +160,31 @@ def resource_name
 
     private
 
+    def deployment_params
+      params.require(:deployment).permit(:name, :description, :deploy_rhev, :deploy_cfme,
```

Comment:
@jmontleon, looks good. similar to the changes that I made in PR https://github.com/fusor/fusor/pull/775 that I will rebase after this is merged.